src/programs/unblur/unblur.cpp

```diff
@@ -854,7 +888,9 @@ bool UnBlurApp::DoCalculation()
 			temp_float[0] = x_shifts[image_counter] * output_pixel_size;
 			temp_float[1] = y_shifts[image_counter] * output_pixel_size;
 			shifts_file.WriteLine(temp_float);
-			wxPrintf("image #%li = %f, %f\n", image_counter, result_array[image_counter], result_array[image_counter + number_of_input_images]);
+      #ifdef PRINT_VERBOSE
```

Comment:
Is `PRINT_VERBOSE` usually defined? How is it different from `DEBUG`?

Reply:
No this is an atypical usage. It is only defined when the extra profiling is enabled. It is a temporary convenience for development to avoid clutter in the timing output. It's different because I want PRINT_VERBOSE to happen in debug mode. 

I'm trying to make my PRs as contained as possible. In the next one (or next next) I'll setup an --enable-timing configure option to get rid of the HYPOTHETICAL_TIMING_DEFINE (or whatever I called it). At the same time, I'll think about how to handle this print output better. 

I already a have a simple script to parse the output, so I guess the PRINT_VERBOSE really is not needed.

Reply:
@jojoelfe looks like I somehow left out the actual define! Must have happened when I cleaned up the commit prior to pushing. Glad you caught this - fix pushed

Reply:
(since I pushed this, the PR needs approval again)